completions/ssh.completion.sh

```diff
@@ -18,6 +20,17 @@ function _omb_completion_ssh {
     COMPREPLY=($(compgen -W "$(grep ^Host "$HOME/.ssh/config" | awk '{for (i=2; i<=NF; i++) print $i}' )" "${options[@]}"))
   fi
 
+  # check if .ssh/config contains Include options
+  for additional_include_option in $(awk -F' ' '/^Include/{print $2}' "$HOME/.ssh/config" 2>/dev/null) ;do
```

Comment:
I think this entire `for`-loop is valid only when `~/.ssh/config` is readable, so I think we can put it inside the previous `if` statement of `[[ -r $HOME/.ssh/config ]]`.

Reply:
f1e0cc0